From 3073c41cb968ad3c92ab0281b026f7cd8f7d1944 Mon Sep 17 00:00:00 2001
From: Michael Schroeder <mls@suse.de>
Date: Wed, 25 Jun 2025 11:39:52 +0200
Subject: [PATCH] Change updateinfo filtering error to a warning

The backend does not give us any updateinfo if there is no package
matching a selection. So it's better to build an empty product instead
of erroring out.

We may want to revisit this in the future.
---
 src/productcomposer/utils/rpmutils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/productcomposer/utils/rpmutils.py b/src/productcomposer/utils/rpmutils.py
index d9ef49f..a028162 100644
--- a/src/productcomposer/utils/rpmutils.py
+++ b/src/productcomposer/utils/rpmutils.py
@@ -161,7 +161,7 @@ def link_rpms_to_tree(rpmdir, yml, pool, arch, flavor, tree_report, supportstatu
     referenced_update_rpms = None
     if 'updateinfo_packages_only' in yml['build_options']:
         if not pool.updateinfos:
-            die("filtering for updates enabled, but no updateinfo found")
+            warn("filtering for updates enabled, but no updateinfo found")
         if singlemode:
             die("filtering for updates enabled, but take_all_available_versions is not set")
 
